calculator_test.go

```diff
@@ -24,31 +24,43 @@ func TestSubtract(t *testing.T) {
 }
 
 type TestCase struct {
-	A    float64
-	B    float64
+	a    float64
+	b    float64
+	f    func(float64, float64) float64
 	want float64
+	name string
```

Comment:
It'll make sense to put the `name` field first, in both the struct definition, and the cases, won't it? And we can use more detailed names, like `Add two positive integers`, or `Multiply a positive fraction by a negative fraction`.